### PR TITLE
app-emulation/libvirt: Version updated to 6.7.0.

### PR DESCRIPTION
**Pull request by jpds**
Tested on a `amd64` system. I took the ebuild for `9999` as upstream had changed their build system and adapted the existing patches.

### CODE REVIEW

File: app-emulation/libvirt/libvirt-6.7.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+	)
+	udev? (
+		virtual/udev
+		>=x11-libs/libpciaccess-0.10.9
+	)
+	zfs? ( sys-fs/zfs )"
+
+DEPEND="${RDEPEND}
+	${PYTHON_DEPS}
+	app-text/xhtml1
```
Comment (Zlogene) [inline app-emulation/libvirt/libvirt-6.7.0.ebuild:118]: these all deps listed below are potential BDEPEND ones.

File: app-emulation/libvirt/libvirt-6.7.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+
+	ERROR_USER_NS="Optional depending on LXC configuration."
+
+	if [[ -n ${CONFIG_CHECK} ]]; then
+		linux-info_pkg_setup
+	fi
+}
+
+src_prepare() {
+	touch "${S}/.mailmap"
```
Comment (Zlogene) [inline app-emulation/libvirt/libvirt-6.7.0.ebuild:216]: `|| die`

File: app-emulation/libvirt/libvirt-6.7.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+	export VIR_TEST_DEBUG=1
+	meson_src_test
+}
+
+src_install() {
+	meson_src_install
+
+	# Remove bogus, empty directories. They are either not used, or
+	# libvirtd is able to create them on demand
+	rm -rf "${D}"/etc/sysconfig
```
Comment (Zlogene) [inline app-emulation/libvirt/libvirt-6.7.0.ebuild:302]: all rm calls also need `|| die`

File: app-emulation/libvirt/libvirt-6.7.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+	sasl? ( dev-libs/cyrus-sasl )
+	selinux? ( >=sys-libs/libselinux-2.0.85 )
+	virt-network? (
+		net-dns/dnsmasq[script]
+		net-firewall/ebtables
+		>=net-firewall/iptables-1.4.10[ipv6]
+		net-misc/radvd
+		sys-apps/iproute2[-minimal]
+	)
+	virtualbox? ( || ( app-emulation/virtualbox >=app-emulation/virtualbox-bin-2.2.0 ) )
```
Comment (Zlogene) [inline app-emulation/libvirt/libvirt-6.7.0.ebuild:104]: Drop virtuaalbox-bin dependency here.

File: app-emulation/libvirt/libvirt-6.7.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+	>=dev-libs/libxml2-2.7.6
+	>=net-analyzer/openbsd-netcat-1.105-r1
+	>=net-libs/gnutls-1.0.25:0=
+	net-libs/libssh2
+	net-libs/libtirpc
+	net-libs/rpcsvc-proto
+	>=net-misc/curl-7.18.0
+	sys-apps/dmidecode
+	>=sys-apps/util-linux-2.17
+	sys-devel/gettext
```
Comment (Zlogene) [inline app-emulation/libvirt/libvirt-6.7.0.ebuild:64]: this is also BDEPEND most-likely.

File: app-emulation/libvirt/libvirt-6.7.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,343 @@
...
+	dev-libs/libnl:3
+	>=dev-libs/libxml2-2.7.6
+	>=net-analyzer/openbsd-netcat-1.105-r1
+	>=net-libs/gnutls-1.0.25:0=
+	net-libs/libssh2
+	net-libs/libtirpc
+	net-libs/rpcsvc-proto
+	>=net-misc/curl-7.18.0
+	sys-apps/dmidecode
+	>=sys-apps/util-linux-2.17
```
Comment (Zlogene) [inline app-emulation/libvirt/libvirt-6.7.0.ebuild:63]: You do not need this dep declared explicitly.

File: app-emulation/libvirt/libvirt-6.7.0.ebuild [hunk 7]
```diff
@@ -0,0 +1,343 @@
...
+		sys-apps/iproute2[-minimal]
+	)
+	virtualbox? ( || ( app-emulation/virtualbox >=app-emulation/virtualbox-bin-2.2.0 ) )
+	wireshark-plugins? ( net-analyzer/wireshark:= )
+	xen? (
+		>=app-emulation/xen-4.6.0
+		app-emulation/xen-tools:=
+	)
+	udev? (
+		virtual/udev
```
Comment (Zlogene) [inline app-emulation/libvirt/libvirt-6.7.0.ebuild:111]: According to https://github.com/libvirt/libvirt/blob/v6.7.0/meson.build#L1401
`virtual/libudev` would be enough.

**Review by Zlogene — CHANGES_REQUESTED** [review]

File: app-emulation/libvirt/libvirt-6.7.0.ebuild (hunk 5 above)
Comment (jpds) [reply app-emulation/libvirt/libvirt-6.7.0.ebuild:71]: `/usr/libexec/libvirt-guests.sh` still seems to call gettext directly as per the comment above.

**Pull request closed without merging**